### PR TITLE
Create FAQ

### PR DESCRIPTION
**Pull request by stephffuller**
SFuller-FAQ

### CODE REVIEW

File: docs/FAQ [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- [Redoc Community Edition is not rendering my large json file smoothly.](#redoc-community-edition-is-not-rendering-my-large-json-file-smoothly)
+- [Does Redocly support webhooks with OpenAPI 3.1?](#does-redocly-support-webhooks-with-openapi-31)
+- [How do I adjust the theme settings on a page, such as color, line spacing, and fonts?](#how-do-i-adjust-the-theme-settings-on-a-page-such-as-color-line-spacing-and-fonts)
+- [Where can I locate Docker images for Redocly and redoc.cli?](#where-can-i-locate-docker-images-for-redocly-and-redoc-cli)
+- [Which tag should I use for operations that cannot be logically grouped?](#which-tag-should-i-use-for-operations-that-cannot-be-logically-grouped)
+- [Which vendor extensions does Redocly support?](#which-vendor-extensions-does-redocly-support)
+
+## How do I enable the Console API?
+
+Redoc.ly API Reference Docs supports the Console API, though it is disabled by default. You can enable it in 'API Reference Settings'->'Live Configuration'.
```
Comment (adamaltman) [inline docs/FAQ:13]: Most people probably aren't aware of our premium product. I think we should say something about it not being available in the community edition?

File: docs/FAQ [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Redoc Community Edition is not rendering my large json file smoothly.
+
+Consider upgrading. Redocly API Reference Docs includes an option to section files.
+
+'referenceDocs:
```
Comment (adamaltman) [inline docs/FAQ:19]: The code sample should be in a fenced code block:

https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

File: docs/FAQ [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+   layout:
+      scope: section`
+
+If upgrading is not an option, simplifying your schema or reducing file size can both help to improve performance.
+
+## Does Redocly support webhooks with OpenAPI 3.1?
+
+Yes, Redocly supports OpenAPI 3.1 webhooks. See [\#1304](https://github.com/Redocly/redoc/pull/1304) for more information.
```
Comment (adamaltman) [inline docs/FAQ:27]: I guess technically it will work only after that PR is merged.

File: docs/FAQ [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Redoc.ly API Reference Docs supports themes, with a limited subset of theme features supported by Redoc Community Edition. Discover more information on configuring page themes [here](https://redoc.ly/docs/cli/configuration/reference-docs/#theming).
+
+## Where can I locate Docker images for Redocly and redoc cli?
+
+[Redoc Docker Image](https://hub.docker.com/r/redocly/redoc)  
+[redoc.cli Docker Image](https://hub.docker.com/r/broothie/redoc-cli). See [\#1158](https://github.com/Redocly/redoc/pull/1158) for more information.  
+
+## Which tag should I use for operations that cannot be logically grouped?
+
+When `x-tagGroups` is enabled, each tag must have a group or it will not be displayed. For operations that cannot be logically grouped, use the `other` group/tag. Lint your API definition to ensure that each path has a tag.
```
Comment (adamaltman) [inline docs/FAQ:40]: Should we let people know how to lint their API definition and which of our built-in rules identifies if an operation isn't tagged?

Also, should we note that there is not a built in rule to check that any tags aren't missing from `x-tagGroups` group definitions?

File: docs/FAQ (hunk 2 above)
Comment (stephffuller) [reply docs/FAQ:19]: Well, that's frustrating. I did add the 4 spaces, but they must have been lost on copy+paste. Readding.

File: docs/FAQ (hunk 3 above)
Comment (stephffuller) [reply docs/FAQ:27]: Yep :)

File: docs/FAQ (hunk 4 above)
Comment (stephffuller) [reply docs/FAQ:40]: Is that this: https://redoc.ly/docs/cli/built-in-rules/#operation-tag-defined ?

Comment (adamaltman) [reply docs/FAQ:40]: Yes, that's it.

File: docs/FAQ (hunk 1 above)
Comment (stephffuller) [reply docs/FAQ]: Done

File: docs/FAQ (hunk 2 above)
Comment (stephffuller) [reply docs/FAQ]: Done

File: docs/FAQ (hunk 4 above)
Comment (stephffuller) [reply docs/FAQ]: Done.

**Comment by skadinna** [comment]
Hello! We're cleaning up old docs-related PRs in this repository. As this one was part of trial day work and it hasn't been merged in the meantime, we're going to close it. You're always welcome to contribute to Redoc by opening a fresh pull request. :)

**Pull request closed without merging**